packages/kit/src/views/Prime/pages/PrimeDashboard/PrimeDashboard.tsx

```
@@ -89,6 +90,7 @@ export default function PrimeDashboard({
 
   const [selectedSubscriptionPeriod, setSelectedSubscriptionPeriod] =
     useState<ISubscriptionPeriod>('P1Y');
+  const [serverUserInfo, setServerUserInfo] = useState<IPrimeServerUserInfo | undefined>(undefined);
```

ESLint

[error] Replace `IPrimeServerUserInfo·|·undefined` with `⏎····IPrimeServerUserInfo·|·undefined⏎··`

(prettier/prettier)